rewrite-java/src/main/java/org/openrewrite/java/style/ImportLayoutStyle.java

```diff
@@ -878,8 +920,26 @@ private void serializeFields(ImportLayoutStyle value, JsonGenerator gen) throws
                 })
                 .toArray(String[]::new);
 
+        @SuppressWarnings("SuspiciousToArrayCall") String[] packagesToWildcard = value.getPackagesToWildcard().stream()
+                .map(block -> {
+                    if (block instanceof ImportLayoutStyle.Block.ImportPackage) {
+                        ImportLayoutStyle.Block.ImportPackage importPackage = (ImportLayoutStyle.Block.ImportPackage) block;
+                        String withSubpackages = importPackage.getPackageWildcard().pattern().contains("[^.]+") ? " without subpackages" : "";
+                        return "import " + (importPackage.isStatic() ? "static " : "") + importPackage.getPackageWildcard().pattern()
+                                .replace("\\.", ".")
+                                .replace(".+", "*")
+                                .replace("[^.]+", "*")  + withSubpackages;
+                    }
+                    return new UnsupportedOperationException("Unknown block type " + block.getClass().getName());

```

Comment:
You probably meant to `throw`, rather than `return` this. Hence the `SuspiciousToArrayCall` warning

Reply:
Posterity:
I returned the exception instead of throwing since it follows the same behavior as serializing the layout section (line 935).
Returning early may change expected behavior, so I'm waiting on input from the team to use throws instead of a return.